blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/TestIO.java

```diff
@@ -0,0 +1,24 @@
+package com.rackspacecloud.blueflood.io;
+
+/**
+ * Utility class to provide some test methods existing tests use.
+ */
+public interface TestIO {
```

Comment:
Another idea would be to create an interface, preferably with a different name, and just have the two MetadataIO classes (Astyanax and Datastax) to implement it. But the getKeyCount() would not take a String parameter for the column name. It is implied that it operates on whatever ColumnFamily that is used to store Metadata.